Libplanet/Net/Swarm.cs

```diff
@@ -61,14 +62,16 @@ public partial class Swarm : ICollection<Peer>, IDisposable
             IPAddress ipAddress = null,
             int? listenPort = null,
             DateTimeOffset? createdAt = null,
-            IEnumerable<IceServer> iceServers = null)
+            IEnumerable<IceServer> iceServers = null,
+            int protocolVersion = 1)
```

Comment:
Since it's not about Libplanet's internals but application's own speak, `appProtocolVersion` or such name might be better.

And I still it's better not to have the default version, as it's prone to be unaware of this.